src/frontend/app.ts

```diff
@@ -241,10 +241,9 @@ export default (octokitApp: App): Express => {
       'Not Found': 404,
     };
 
-    const errorStatusCode = errorCodes[err.message] || 400
+    const errorStatusCode = errorCodes[err.message] || 500
```

Comment:
If we were unable to resolve the error code by exception, we'll return 500 status code, hence the error is unknown. ( We can't blame the client if we not sure 🙂 )